test/unit/search-test.js

```diff
@@ -78,7 +78,7 @@ describe('Search', () => {
     let directCount;
 
     it('should search by owner', async () => {
-      let query  = {owner: 'milesm'};
+      let query  = {owner: 'mercert'};
```

Comment:
you don't have to use Thomas' id. In fact it might be better to use one of our test ids.

Reply:
As discussed - will leave as mercert, since our test ids do not currently have ownership